.github/workflows/deploy-standalone-plugins.yml

```
@@ -50,7 +62,7 @@ jobs:
             # for use in the matrix.
             # The "dry-run" parameter is included here to set the deployment mode.
             # When running the manual (workflow_dispatch) workflow, this value will be set from manual input type.
-            echo "matrix="$(jq -c '{include:[keys[] as $k | {name:$k,slug:.[$k].slug,version:.[$k].version,"dry-run":false }]}' plugins.json) >> $GITHUB_OUTPUT
+            echo "matrix="$(jq -c '{include:[.modules | to_entries[] | {name:.key,slug:.value.slug,version:.value.version,"dry-run":true }]}' plugins.json) >> $GITHUB_OUTPUT
```

```suggestion
            echo "matrix="$(jq -c '{include:[.modules | to_entries[] | {name:.key,slug:.value.slug,version:.value.version,"dry-run": false }]}' plugins.json) >> $GITHUB_OUTPUT
```

Once we complete testing we will make it `false` for auto workflow.